chart/kubeapps/values.schema.json

```diff
@@ -1,135 +1,159 @@
-          "hidden": {
-            "value": false,
-            "path": "ingress/enabled"
-          }
-        },
-        "certManager": {
-          "type": "boolean",
           "form": true,
-          "title": "Use cert-manager to auto-generate the TLS certificate",
-          "description": "Add the corresponding annotations for cert-manager to auto-generate the TLS certificate",
           "hidden": {
-            "value": false,
-            "path": "ingress/enabled"
-          }
+            "path": "ingress/enabled",
+            "value": false
+          },
+          "title": "Enable TLS configuration",
+          "type": "boolean"
         }
-      }
+      },
+      "title": "Ingress configuration"
     },
-    "authProxy": {
-      "type": "object",
-      "title": "OIDC Proxy configuration",
+    "securityContext": {
+      "form": true,
       "properties": {
         "enabled": {
-          "type": "boolean",
           "form": true,
-          "title": "Enable OIDC proxy",
-          "description": "Use an OIDC provider in order to manage accounts, groups and roles with a single application"
-        },
-        "provider": {
-          "type": "string",
-          "form": true,
-          "title": "Identity Provider name",
-          "description": "See https://pusher.github.io/oauth2_proxy/auth-configuration to find available providers",
-          "hidden": {
-            "value": false,
-            "path": "authProxy/enabled"
-          }
-        },
-        "clientID": {
-          "type": "string",
-          "form": true,
-          "title": "Client ID:",
-          "description": "Client ID of the Identity Provider",
-          "hidden": {
-            "value": false,
-            "path": "authProxy/enabled"
-          }
+          "title": "Enable security context",
+          "type": "boolean",
+         "default": false
         },
-        "clientSecret": {
-          "type": "string",
+        "fsgroup": {
           "form": true,
-          "title": "Client Secret",
-          "description": "Secret used to validate the Client ID",
           "hidden": {
-            "value": false,
-            "path": "authProxy/enabled"
-          }
+            "path": "securityContext/enabled",
+            "value": false
+          },
+          "title": "File System Group ID",
+          "type": "integer"
         },
-        "cookieSecret": {
-          "type": "string",
+        "runAsUser": {
           "form": true,
-          "title": "Cookie Secret",
-          "description": "Used by OAuth2 Proxy to encrypt any credentials",
           "hidden": {
-            "value": false,
-            "path": "authProxy/enabled"
-          }
+            "path": "securityContext/enabled",
+            "value": false
+          },
+          "title": "File System User ID",
+          "type": "integer"
         }
-      }
+      },
+      "title": "Frontend configuration"
     }
   }
-}
+}
```

Comment:
missing empty line at the end

Reply:
Ok. I suggest to include enforcement for formatting, such as prettier.